### DISCUSSION TITLE
How to remove browser version in order to get the same visitorId even if the version changes?

### DISCUSSION THREAD

**Discussion by bergmp** [fingerprintjs/fingerprintjs#736] [Q&A] [answered]
I am trying to identify users with fingerprintjs. However, when browser version changes visitorId changes but the user is the same.
How to solve that?

**Comment by makma** [comment, marked as answer]
Hi @bergmp!
The specific browser version is not an entropy component (or signal) for the `visitorId`. However, it's plausible, an update of the browser can change some entropy sources. To identify which entropy component affects the final `visitorId` I'd recommend comparing the `result` object. Then you can easily [exclude the specific entropy source](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/extending.md#extending-fingerprintjs). 
```js
// ...

const result = await fp.get()

// The `languages` and `audio` components will be excluded
const { languages, audio, ...components } = result.components

// Optionally, you can make a visitor identifier from your custom list of components
const visitorId = FingerprintJS.hashComponents(components)
```
Nevertheless, please note, by excluding some entropy sources you might affect sensitivity (the number of unique fingerprints).

**Reply by bergmp** [reply]
Thanks for your support.
